setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml

```diff
@@ -0,0 +1,51 @@
+# Uses apt-get to install system-wide dependencies and pip to install
+# python dependencies.
+
+# If using Ubuntu 13.04, you may discover that some repositories are missing.
+# See this for details:
+# https://gist.github.com/dergachev/f5da514802fcbbb441a1
+- name: Handle Ubuntu 13.04 end of life.
+  sudo: True
+  shell: sed -i.bak -r 's/(archive|security).ubuntu.com/old-releases.ubuntu.com/g' /etc/apt/sources.list
+  when: ansible_distribution == 'Ubuntu' and ansible_distribution_version == '13.04'
+
+- name: Install base Ubuntu packages
+  apt: pkg={{ item }} state=installed update_cache=yes
+  sudo: True
+  with_items:
+    - git
```

Comment:
I think git is required at least for doing a ```git describe``` when building the console binaries.